package.json

```diff
@@ -6,6 +6,38 @@
   "author": "NebulousLabs",
   "description": "A UI application for interfacing with Sia",
   "license": "MIT",
+  "scripts": {
+    "debug": "webpack -d && electron --remote-debugging-port=9222 .",
+    "format": "prettier-standard 'js/**/*.js' 'plugins/**/*.js'",
+    "build-production": "webpack --config ./webpack.production.config.js",
+    "watch": "webpack --watch",
+    "clean": "rm -rf release doc/Sia-UI node_modules Sia config.json **/*.swp npm-debug.log",
+    "fresh": "npm run clean && npm install && npm start",
+    "test-args": "BABEL_ENV=test mocha --require babel-polyfill --require test/dom.setup.js --compilers js:babel-register --recursive ./test --exit",
+    "test": "npm run lint && npm run build && npm run test-args",
+    "precommit": "lint-staged",
+    "start": "cross-env NODE_ENV=production electron .",
+    "dev": "cross-env START_HOT=1 node -r babel-register ./internals/scripts/CheckPortInUse.js && cross-env START_HOT=1 npm run start-renderer-dev",
+    "start-renderer-dev": "cross-env NODE_ENV=development node --trace-warnings -r babel-register ./node_modules/webpack-dev-server/bin/webpack-dev-server --config ./internals/webpack/webpack.config.renderer.dev.js",
+    "start-main-dev": "cross-env HOT=1 NODE_ENV=development electron -r babel-register ./app.js",
+    "build": "rimraf dist && concurrently \"npm run build-main\" \"npm run build-renderer\"",
+    "build-main": "cross-env NODE_ENV=production node --trace-warnings -r babel-register ./node_modules/webpack/bin/webpack --config ./internals/webpack/webpack.config.main.prod.js --colors",
+    "build-renderer": "cross-env NODE_ENV=production node --trace-warnings -r babel-register ./node_modules/webpack/bin/webpack --config ./internals/webpack/webpack.config.renderer.prod.js --colors",
+    "lint": "standard --max-warnings 0 ./index.js ./plugins ./js ./test",
+    "lint-fix": "standard --fix ./index.js ./plugins ./js ./test"
+  },
+  "lint-staged": {
+    "linters": {
+      "js/**/*.js": [
+        "prettier-standard",
+        "git add"
+      ],
+      "plugins/**/*.js": [
+        "prettier-standard",
+        "git add"
```

Comment:
seems weird to me that `git` would be called directly from package.json. I'd rather have this command just do the linting, and let the user call `git add -u` separately.